uint8 axis
float32 traj_vel_limit
---
bool success